src/js/control-bar/time-controls/time-display.js

```diff
@@ -89,7 +89,7 @@ class TimeDisplay extends Component {
 
       let oldNode = this.textNode_;
 
-      if (oldNode && !this.contentEl_.contains(oldNode)) {
+      if (oldNode && this.contentEl_.firstChild !== oldNode) {
```

Comment:
Should we check `oldNode.parentElement` and `oldNode.parentElement === this.contentEl_` that way we don't get into a situation where `contentEl_` has multiple elements and the first one is not oldNode, causing us to insert nodes over and over.

Reply:
Maaaybe. Time displays should only have the one text node in them.